src/Models/AuthenticationLogRecord.php

```diff
@@ -20,11 +20,27 @@ public function authenticatable()
         return $this->morphTo();
     }
 
-    public static function createWithOmissions(array $data)
+    public static function createWithConfigFilters(array $data)
     {
-        if(array_key_exists('credentials', $data)) {
-            $data['credentials'] = Omissions::omitCredentials($data['credentials']);
+        // TODO should find a nicer way to write the to avoid copy and pasting
```

Comment:
Be careful of TODOs -- they have a tendency to hang around in packages untouched for a long time and then end up getting abandoned - I try and pass through my code and resolve all TODOs before my final PRs, but leaving them there while you are working on code is fine as long as you plan to come back and resolve them - Ideally put a ticket in the project on Jira so that we have a record of the todo somewhere other than the codebase.